channels/af.m3u

```diff
@@ -1,21 +1,15 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/x8mlbJM.jpg" group-title="",Ariana News
-https://d10rltuy0iweup.cloudfront.net/ATNNEWS/myStream/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/x8mlbJM.jpg" group-title="News",Ariana News
 https://d10rltuy0iweup.cloudfront.net/ATNNEWS/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV
-https://d10rltuy0iweup.cloudfront.net/ATNNAT/myStream/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV
 https://d10rltuy0iweup.cloudfront.net/ATNNAT/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV
-https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV US
-https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Usbeki" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV (Usbeki)
```

Comment:
Keep channel name as Ariana TV, keep language as Pushto.

Reply:
The language used is mentioned on Ariana TV's website. The US mention is Usbeki, not Pushto.

Reply:
Usbeki is not an iso 639-3 language, Uzbek is. However I just listened to the stream, its definitely broadcasting in Pushto.